src/items/class-features/bedside_manner_(ex).json

```diff
@@ -0,0 +1,115 @@
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "none",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>&nbsp;@Icon[type:language-dependent] Language-Dependent</p>\n<p>&nbsp;@Icon[type:mind-affecting] Mind-Affecting</p>\n<p>@Icon[type:sense-dependent] Sense-Dependent</p>\n<p>When you make a Medicine check to treat deadly wounds or successfully aid another for another creature’s Medicine check to treat deadly wounds, if the Medicine check is successful you can add your Charisma bonus to the number of Hit Points recovered as a result of the skill check. Additionally, when you make a Medicine check to treat disease or treat drugs or poison, or successfully aid another for another creature’s Medicine check to do so, if the Medicine check is successful, the saving throw bonus granted by the skill checks increases to +6. If the associated saving throw succeeds, it counts as two consecutive successes for the purposes of curing the intended affliction.</p>"
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": true,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": true,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": true,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "duration": {
+      "units": "",
+      "value": ""
+    },
+    "formula": "",
+    "isActive": null,
+    "modifiers": [],
+    "range": {
+      "additional": "",
+      "per": "",
+      "units": "none",
+      "value": null
+    },
+    "recharge": {
+      "charged": false,
+      "value": null
+    },
+    "requirements": "6th Level (Envoy)",
+    "rollNotes": "",
+    "save": {
+      "type": "",
+      "dc": "",
+      "descriptor": "negate"
+    },
+    "source": "AR, pg. 144",
```

Comment:
No commas between the book initials and `pg`. Cook should be shouting at you about this

Reply:
Cook isn't saying boo about it....  Will go back thru and fix

Reply:
Fixed